dbms/tests/integration/image/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN apt-get update && env DEBIAN_FRONTEND=noninteractive apt-get install --yes -
     tzdata \
     libreadline-dev \
     libicu-dev \
+    bsdutils \
```

Comment:
Dependency is also need to be added to the `clickhouse-test` package.

Reply:
Done